pkg/endpoints/utils.go

```diff
@@ -107,3 +112,45 @@ func (r Resource) RegisterReadinessProbes(container *restful.Container) {
 
 	container.Add(wsv4)
 }
+
+// Store all types here that are reused throughout files
+type Extension struct {
+	Name string
+	Url  string
+	Port string
+}
+
+
+func (r Resource) RegisterExtension(container *restful.Container, namespace string) {
+	logging.Log.Info("Adding API for extensions")
+	svcs, err := r.K8sClient.CoreV1().Services(namespace).List(metav1.ListOptions{LabelSelector: "tekton-dashboard-extension=true"})
```

Comment:
As this is a super important string (`tekton-dashboard-extension`) let's get it declared upfront as a const somewhere easy to spot, and well documented